server/fishtest/rundb.py

```diff
@@ -381,7 +383,17 @@ def get_run(self, r_id):
                     }
                 return run
         else:
-            return self.runs.find_one({"_id": ObjectId(r_id)})
+            result = requests.get(
+                f"https://localhost/api/serialize_run/{r_id}", verify=False
```

Comment:
http with localhost

Reply:
Does nginx pass through http requests?

Reply:
Only /api, already setup on DEV the allow and deny directive for that URL.
https requires a domain name.

Reply:
Changed it to http.